cmd/main/main.go

```diff
@@ -45,14 +47,14 @@ func main() {
 		}
 
 		logger.Printf("Received message with method: %s", method)
-		handleMessage(logger, method, contents)
+		handleMessage(logger, method, contents, stateMap)
 	}
 }
 
-func handleMessage(logger *log.Logger, method string, contents []byte) {
+func handleMessage(logger *log.Logger, method string, contents []byte, stateMap map[string]int) {
```

Comment:
Is the stateMap concerned with anything other than the state of the currently open file?